In [1]:
import pandas as pd
import numpy as np
import seaborn as sns
import os

%matplotlib inline
import matplotlib.pyplot as plt
                    

In [3]:
df = pd.read_csv('HireArt.csv')

In [4]:
df.head()

Account manager                 Client Name Date of Contact
0     Milan Crona     Wyman, Farrell and Haag      2015-10-31
1    Aidan Pouros  Veum, McClure and Schuster      2013-12-15
2    Aidan Pouros  Veum, McClure and Schuster      2013-12-07
3     Milan Crona             Armstrong Group      2013-11-02
4    Aidan Pouros   Lueilwitz, Moore and Hahn      2013-12-25

Checking for null value in data. We run the code below 

In [5]:
df.isnull().values.any()

False

In [6]:
df.describe()

Account manager        Client Name Date of Contact
count             1001               1001            1001
unique               4                 35             704
top        Milan Crona  Eichmann and Sons      2013-10-11
freq               286                 38               7

### Exploring the dataframe above, we can conclude that 
1. The total count for contacts made is 1001
2. There are 704 unique  date of contacts
3. There are 4 unique managers who made contacts
4. There are 35 unique clients contacted.
5. The maximum appearing top manager is "Milan Crona"
6. "Milan Crona" made 286 contacts with clients
7. The maximun appearing client name is "Eichmann and sons"
8. "Eichmann and sons" was contacted 38 times.
9. The day, month and year with the most contacts is October 11th, 2013 (2013-10-11)
10. The month the team are likely to contact the most clients is October.

